pulsar-functions/instance/src/main/python/log.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
ASF headers looks repeated twice in some Python files

Reply:
nice catch. will try to fix it or do it in a separate PR